tests/utils.py

```diff
@@ -0,0 +1,51 @@
+import string
+import time
+from random import choice, randint
+
+from matrix_client.errors import MatrixRequestError
+
+from raiden.network.transport.matrix.client import GMatrixHttpApi
+from raiden.tests.mocked.test_matrix_transport import \
+    create_new_users_for_address
+
+
+class TestGMatrixHttpApi(GMatrixHttpApi):
```

Comment:
If you start the class name with `Test`, test runners will pick this up as a test class. This won't do much unless you also add a `test_something` method, but it might be irritating in some way.